src/components/UserInfo.js

```diff
@@ -16,6 +17,11 @@ class UserInfo {
     this._titleElement.textContent = title;
     this._descriptionElement.textContent = description;
   }
+
+  // new avatar
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a separate method for updating the avatar, since it's controlled by a different form